packages/components/src/horizontal-bar-list/horizontal-bar-grid-item.tsx

```diff
@@ -70,14 +70,14 @@ const HorizontalBarListItem = ( {
 			<>
 				{ label.length > 1
 					? label.map( ( item ) => (
-							<>
+							<Fragment key={ item.labelIcon }>
```

Comment:
Just curious if the `labelIcon` would be duplicated from icons.

Reply:
Good point! Not likely but they could be. Addressed in f09b5d8054f